owncloudApp/src/main/java/com/owncloud/android/presentation/releasenotes/ReleaseNotesViewModel.kt

```diff
@@ -55,6 +55,11 @@ class ReleaseNotesViewModel(
                 subtitle = R.string.release_notes_bugfixes_subtitle,
                 type = ReleaseNoteType.BUGFIX
             ),
+            ReleaseNote(
+                title = R.string.release_notes_4_4_0_title_improved_from_original_folder_auto_upload,
+                subtitle = R.string.release_notes_4_4_0_subtitle_improved_from_original_folder_auto_upload,
+                type = ReleaseNoteType.ENHANCEMENT
+            ),
```

Comment:
This feature is the main feature of the beta. Indeed, is the only one that is important. It should be the first one, on the top of the release notes. Setting in on the bottom gives it less importance than the ones on the top.

Reply:
Yes, right.